### ISSUE TITLE
method for getting inner value for Term?

### ISSUE THREAD

**Issue by joepio** [oxigraph/rio#61]
Hi there! Thanks for creating / maintaining this.

I'm currently parsing some turtle documents, and I need to get the plain value inside the terms. The `to_string()` method adds N-Triples serialization syntax, which often makes sense, but I'm looking for the inner value.

I'm assuming there must be some way for me to get the value from a Term in some easy way. At the moment, the code taht does this looks like this:

```rust 
// Returns the innver value of a Term in an RDF triple. If it's a blanknode or triple inside a triple, it will return None.
use rio_api::model::Term;
fn get_inner_value(t: Term) -> String {
    match t {
        Term::Literal(lit) => match lit {
            rio_api::model::Literal::Simple { value } => value.into(),
            rio_api::model::Literal::LanguageTaggedString { value, language: _ } => {
                value.into()
            }
            rio_api::model::Literal::Typed { value, datatype: _ } => value.into(),
        },
        Term::NamedNode(nn) => nn.iri.into(),
        // etc.
    }
}

```

**Comment by Tpt** [comment]
Hi! Thank you for using Rio.

That's a great question. I wrote Rio as a library underlying more user friendly libraries like [Oxigraph](https://github.com/oxigraph/oxigraph) or [Sophia](https://github.com/pchampin/sophia_rs). I did not spent time making the API very convenient to use.

Anyway, providing an accessor to the term "value" in general is a non-trivial thing. For blank node should we return the ID or None like your function does? What about nested triples? I would tend to believe each API user is likely to want a specific behavior and having to write ~10 LOCs to define which behavior the user exactly want is safer than providing a generic function that is likely to not behave exactly as the user want.
